### DISCUSSION TITLE
Soft launch activities

### DISCUSSION THREAD

**Discussion by Truthans** [unikubehq/marketing#24] [Strategy & Ideas]
I'd like to discuss our potential activities for the upcoming soft launch. I know, we don't want to play it too hard yet but some activities may make sense. Once we now what we want to focus on, we can create the issues.

Potential things I see:

- Mail outreach to some of the companies we talked to during the last weeks
- Outreach to our network contacts
- Post with demo video on LinkedIn (also in relevant groups)
- Live demo / webinar
- Blogpost on blueshoe.de
- Post on product hunt
- Highly targeted campaigns (adwords, display)
- Post on reddit
- Campaigns on LinkedIn

What do you guys think @Schille @SteinRobert?

**Comment by Schille** [comment]
I'd say let's get back with the interested people we already talked to in the past. Maybe around 5 appointments should be doable during the next weeks. We can present the basic unikube workflow:
1) creating an unikube Deck (see the buzzword-counter charts)
2) create a simple application running this Deck
3) add everything to the unikube platform: project, deck, environment configuration / maybe a sops PGP key
4) check out the unikube CLI and provision a local K8s cluster
5) switch the application, change code and see it running in K8s right away.

- questions and feedback is warmly welcomed.

**Reply by Schille** [reply]
In addition we should create additional figures and documentation to support the process of digesting the presentation afterwards; specific requests and call-to-actions. What are our expectations for them to do next?
